.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules/
```

Comment:
Should this be /node_modules or ./node_modules ?

Reply:
/node_modules/ worked for me, and I referred this article : https://www.atlassian.com/git/tutorials/gitignore
and it seemed alright to me.
 @anmolbabu @miteshvp @dev-gaur  Correct me if I am wrong

Reply:
Ack